vies.go

```diff
@@ -0,0 +1,68 @@
+package gobltin
```

Comment:
I'd be tempted to consider putting these implementations inside their own internal packages rather than in the main namespace. That'll scale better as potentially more implementations are added.